cmd/cli/handler_client.go

```diff
@@ -56,6 +56,11 @@ func (h *ClientHandler) ImportClients(cmd *cobra.Command, args []string) {
 	cmdx.MinArgs(cmd, args, 1)
 	m := h.newClientManager(cmd)
 
+	ek, encryptSecret, err := newEncryptionKey(cmd, nil)
+	if err != nil {
```

Comment:
`cmdx.Must` will automatically skip if `err == nil`, so you can leave out the `if` check!